tools/postgres/docker-compose.yaml

```diff
@@ -10,4 +10,4 @@ services:
     ports:
       - ${KUMA_STORE_POSTGRES_PORT:-15432}:5432
     volumes:
-      - ../../install/postgres/resource.sql:/docker-entrypoint-initdb.d/init.sql
\ No newline at end of file
+      - ../../app/kumactl/data/install/universal/control-plane/postgres/resource.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
if we had expected output of `kumactl install postgres-schema` in a `testdata` file, I would suggest to use it instead

Reply:
Why? I'd expect for it to use production file.

Reply:
original schema files might include placeholders for variable expansion (e.g., DB name, DB Schema name, DB Owner account name, DB User account name, user passwords, etc), while `*.golden.sql` file has all placeholders resolved

Reply:
Ok, changed.